spec/06-expressions.md

```diff
@@ -1746,7 +1746,7 @@ a = scala.Any
 
 so `scala.Any` is the type inferred for `a`.
 
-### Eta Expansion
+### <a name="eta-expansion-section">Eta Expansion</a>
```

Comment:
!

![well that was obvious](https://media1.giphy.com/media/5nmNFk6tJHGMhEWtxM/giphy.gif?cid=19f5b51a5cdaa15e5865537467a95cb6&rid=giphy.gif)

Reply:
Apparently, it's implementation specific, so we'll see if it works.

https://stackoverflow.com/a/6494918/463761

Reply:
As the kids say, 🤞.